### ISSUE TITLE
ModuleNotFoundError: No module named 'cyberbattle.agents.baseline'

### ISSUE THREAD

**Issue by jonabox** [microsoft/CyberBattleSim#33]
ran init.sh but I'm getting the following error:

``
  File "app.py", line 6, in <module>
    import cyberbattle.simulation.model as model
  File "<frozen zipimport>", line 259, in load_module
  File "/usr/local/lib/python3.8/dist-packages/loonshot_sim-0.1-py3.8.egg/cyberbattle/__init__.py", line 9, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/usr/local/lib/python3.8/dist-packages/loonshot_sim-0.1-py3.8.egg/cyberbattle/agents/__init__.py", line 9, in <module>
ModuleNotFoundError: No module named 'cyberbattle.agents.baseline'
``

I'm running on Ubuntu 18.04, not sure what's going on or what I should do.

**Comment by blumu** [comment]
@jonabox Could you share the entire script output? I am wondering if there is any other error prior to this one leading to it. Also are you running it using system-wide python packages or using a virtual environment?

**Comment by jonabox** [comment]
@blumu Unfortunately, I have since fixed it and can't share the output anymore. I was using the python virtual enviroment and was able to complete the setup by manually going through the commands in init.sh after running `$source venv/bin/activate` within the main directory.

This ocassionaly happens when I set up the repository, which I've done several times now on my Ubuntu machines. I'll make sure to update if it happens again, but for now, I'll close this issue.
